### PR TITLE
Pxc operator 1.9.1 to 1.12.2 + minor subhchart updates

### PR DESCRIPTION
**Pull request by Jancis**
- pxc operator 1.9.1 -> 1.12.2
- ingress-nginx 4.6.0 -> 4.6.1
- nginx-ingress 0.17.0 -> 0.17.1
- instana-agent 1.2.56 -> 1.2.60

### CODE REVIEW

File: silta-cluster/docs/Upgrading-to-0.2.43.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Upgrading silta-cluster chart to 0.2.43
+
+## pxc-operator upgrade (run before upgrading silta-cluster chart)
+
+1. Upgrade CRD's:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/percona/percona-helm-charts/dcfc35a1158862da60a89010e4cabaa2b94560f5/charts/ps-operator/crds/crd.yaml
```
[the remark this answers is not in the document]
Comment (Jancis) [reply silta-cluster/docs/Upgrading-to-0.2.43.md:8]: Files are identical, but i changed it so it's similar. Should be good now.

[the remark this answers is not in the document]
Comment (Rade333) [reply silta-cluster/docs/Upgrading-to-0.2.43.md]: The commit part is good now yes, but are you intentionally pointing to _ps_ operator instead of _pxc_?

**Review by Rade333 — COMMENTED** [review]
The commit part is good now yes, but are you intentionally pointing to ps operator instead of pxc?

**Comment by Jancis** [comment]
No, this is a classic typo and a facepalm moment ;) 
I'll fix it after the current task, no rush.

**Comment by Jancis** [comment]
url updated

**Review by Rade333 — APPROVED** [review]

**Pull request merged**